### DISCUSSION TITLE
A way to disable the number / count badges on the panels?

### DISCUSSION THREAD

**Discussion by krc** [mbnuqw/sidebery#1181] [Ideas]
Talking about the number of open tabs (or number of bookmarks) count on each panel icon. Might be nice to have a toggle to turn those on/off, either per panel or even just globally. Mostly they are fine and even useful, but sometimes my brain decides to get bugged by them. They look like "unread badges" which *really* bother me and I always turn off if possible. These obviously aren't that though, so I guess that is why my brain usually accepts them here.

**Comment by albino1** [comment]
You can turn off the number on the panels here:

`Settings -> Navigation Bar -> Show count of tabs/bookmarks`

HTH.

**Reply by krc** [reply]
Oh thanks! totally could not find that
